core/src/main/scala/org/ensime/core/javac/JavaCompletion.scala

```diff
@@ -209,7 +210,7 @@ trait JavaCompletion extends Helpers with SLF4JLogging {
     caseSense: Boolean
   ): List[CompletionInfo] = {
 
-    typeElement(info, target).map {
+    typeElement(info, target).toList.flatMap {
```

Comment:
Not a gotcha, I was honestly wondering how can this be done nicely, but unfortunately this only works one way: `List(1).flatMap(x => Option(x))`, either way I'm pretty sure that `Option.toList` is super-cheap, so shouldn't really be a problem here.